docassemble/ALAutomatedTestingTests/data/questions/al_set_up_testing.yml

```diff
@@ -86,7 +92,20 @@ code: |
   installer.set_da_info()
   set_da_info = True
 ---
-# github info
+# TODO: Add instructions on how to find the GitHub repo url on Packages page (maybe point to docs)
+id: github auth
+question: |
+  GitHub authorization
+subquestion: |
+  In order to change files in your GitHub repository, we need permission. If you have permission to change the files of the repository, you can give us permission too.
+  
+  To do that, create a [personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token). You can delete the token any time after confirming the setup has worked. You can also use a new one anytime.
```

Comment:
Mention that the token must have repo permissions? (I think you do this later, but I think that information is required here too, right?)

Reply:
Good idea. I'll update that.